lib/va_profile/profile_information/transaction_response.rb

```diff
@@ -0,0 +1,194 @@
+# frozen_string_literal: true
+
+require 'va_profile/models/transaction'
+require 'va_profile/response'
+
+module VAProfile
```

Comment:
The classes defined here also use a duck typing pattern, so maybe we want to add some comments around the required duck typing methods that are assumed and being used elsewhere